cudapoa/include/claragenomics/cudapoa/batch.hpp

```diff
@@ -56,21 +56,23 @@ struct BatchSize
     int32_t max_sequence_size;
     /// Maximum size of final consensus
     int32_t max_concensus_size;
-    /// Maximum number of nodes in a graph, 1 graph per window
+    /// Maximum number of nodes in a POA graph, one graph per window
     int32_t max_nodes_per_window;
-    /// Maximum number of nodes in a graph, 1 graph per window in banded mode
+    /// Maximum number of nodes in a POA graph in banded alignment, one graph per window
     int32_t max_nodes_per_window_banded;
-    /// Maximum vertical dimension of scoring matrix, which stores graph
+    /// Maximum vertical dimension of scoring matrix, which stores POA graph
     int32_t max_matrix_graph_dimension = max_nodes_per_window;
-    /// Maximum vertical dimension of scoring matrix, which stores graph
+    /// Maximum vertical dimension of scoring matrix in banded alignment, which stores POA graph
     int32_t max_matrix_graph_dimension_banded = max_nodes_per_window_banded;
     /// Maximum horizontal dimension of scoring matrix, which stores sequences
     int32_t max_matrix_sequence_dimension = max_sequence_size;
+    /// Bandwidth used in banded alignment
+    int32_t alignment_bandwidth;
```

Comment:
can we rename this to `alignment_band_width`? it's a cosmetic change, but since they differ sementically from the signal processing use of `bandwidth` I just wanted to make that distinction :)

Reply:
sure, done.